gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1,38 +1,53 @@
 GB_HIDDEN_ITEMS_KEY = portal.user.id + "#gradebook#hiddenitems";
 
-GbGradeTable = {};
+GbGradeTable = { _onReadyCallbacks: [] };
 
-let hiddenItems = JSON.parse(sessionStorage.getItem(GB_HIDDEN_ITEMS_KEY));
-
-GbGradeTable._onReadyCallbacks = [];
-
-var addHiddenGbItemsCallback = function () {
+var addHiddenGbItemsCallback = function (hiddenItems) {
 
   GbGradeTable._onReadyCallbacks.push(function () {
 
     hiddenItems.forEach(i => {
-      $(".gb-filter :input:checked[value='" + i + "']").trigger("click", [true]);
+
+      $(".gb-filter :input:checked[value='" + i + "']")
+        .attr("data-suppress-update-view-preferences", "true")
```

Comment:
is this an attribute or a property?

Reply:
I think you usually want to use .prop here instead of .attr:

https://stackoverflow.com/questions/5874652/prop-vs-attr

Reply:
attribute, data attribute. Good way of passing data between js functions.

Reply:
it ends up as element.dataset.suppressUpdateViewPreferences